### ISSUE TITLE
TypeError: list indices must be integers or slices, not str

### ISSUE THREAD

**Issue by Leozyc-waseda** [Xharlie/BtcDet#35]
It seems that this repo has many bugs, such as this dbinfos, there is no 'kitti_dbinfos_val.pkl' under the kitti dataset．
So this error will be reported, has anyone encountered the same error?

```
  File "/home/algo-3/work/BtcDet/btcdet/datasets/multifindbestfit.py", line 38, in extract_allpnts
    all_db_infos = pickle.load(f)[type]
TypeError: list indices must be integers or slices, not str
```

https://github.com/Xharlie/BtcDet/blob/80bee34f2f40931600f812a6edbcb27e51cb7ec3/btcdet/datasets/multifindbestfit.py#L33

**Comment by DheerajYarlagadda** [comment]
Yes, I had came across same error message, Just remove the 'val' from the list in line 475 of [BtcDet/btcdet/datasets/multifindbestfit.py](https://github.com/Xharlie/BtcDet/blob/80bee34f2f40931600f812a6edbcb27e51cb7ec3/btcdet/datasets/multifindbestfit.py#L33)
